system/keymaps/remote.xml

```diff
@@ -577,16 +577,28 @@
     <remote>
       <left>StepBack</left>
       <right>StepForward</right>
-      <up>ChannelUp</up>
-      <down>ChannelDown</down>
+      <up>Up</up>
+      <down>Down</down>
+      <skipplus>SkipNext</skipplus>
+      <skipminus>SkipPrevious</skipminus>
+      <pageup>ChannelUp</pageup>
+      <pagedown>ChannelDown</pagedown>
+      <channelplus>ChannelUp</channelplus>
+      <channelminus>ChannelDown</channelminus>
```

Comment:
@xhaggi 

why not? lot of ir remotes have channel up/down buttons. for linux-input-layer we do map KEY_CHANNELDOWN and KEY_CHANNELUP already.

so we map sometihing in lircmap.xml, that we dont use at all ? 

my 2 cents on this. slap me if you feel I am wrong ;)